src/server/v2.0/handler/scan_all.go

```diff
@@ -54,6 +54,28 @@ func (s *scanAllAPI) Prepare(ctx context.Context, operation string, params inter
 	return nil
 }
 
+// StopScanAll stops the execution of scan all artifacts.
+func (s *scanAllAPI) StopScanAll(ctx context.Context, params operation.StopScanAllParams) middleware.Responder {
+	if err := s.requireAccess(ctx, rbac.ActionDelete); err != nil {
+		return s.SendError(ctx, err)
+	}
+
+	execution, err := s.getLatestScanAllExecution(ctx)
+	if err != nil {
+		return s.SendError(ctx, err)
+	}
+	if execution == nil {
+		message := fmt.Sprintf("no scan all job is found currently")
+		return s.SendError(ctx, errors.BadRequestError(nil).WithMessage(message))
+	}
+	err = s.execMgr.Stop(ctx, execution.ID)
```

Comment:
did you test the large amount artifacts scenario? If there are too many taskes to be cancelled, the API can easily time out.

If the http is timeout, the handler will be break? What about the un-stopped task, art they continue scanning in the backend? Is it possible to continue stop them?